### ISSUE TITLE
Reverse vs Traditional Symbolic Links

### ISSUE THREAD

**Issue by gmsiv4** [SickChill/SickChill#6541]
Quick question as the Wiki does not appear to address this, what is the difference between a Symbolic link and a Symbolic Link Reversed in the post processing options?

Does a Symbolic link leave the file in the download directory and create a symbolic link file in the Library? Or is that the reversed process? 

Ideally I would like the files to be moved to my show Library with symbolic links left in the download directory but can't clarify if this should be the Symbolic link or Symbolic link Reversed. Hardlinks will not work across different file systems so I need to use symbolic links. 

Thanks

**Comment by miigotu** [comment]
Reverse symlink first copies and then makes a link to it in the download folder. Make sure your download client supports still seeding through a symlink though.

**Comment by gmsiv4** [comment]
Hi miigotu, I changed the setting to Reverse SymLinks as that is my desired function, but over the last few months, I have checked and SickChill is not moving the files to the TV directory set. The files remain in the Download location and SickChill shows them as downloaded in Library, but no files are moved to the Library location so other programs (Kodi/Plex) cannot access the files since they are not in Library location. 

I do not have any log errors relating to this, the post process function will say its completed successfully but no files are moved.
